packages/commerce-sdk-react/src/hooks/useAccessToken.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import useAuthContext from './useAuthContext'
+import useLocalStorage from './useLocalStorage'
+import useConfig from './useConfig'
+
+const onClient = typeof window !== 'undefined'
+
+/**
+ * TBD
+ *
+ */
+const useAccessToken = (): string | null => {
+    if (onClient) {
+        const config = useConfig()
+        return useLocalStorage(`${config.siteId}_access_token`)
+    }
+    const auth = useAuthContext()
+    return auth.get('access_token')
+}
```

Comment:
1. The `onClient` check doesn't seem necessary, as that's already built in to `auth.get`.
2. All of these hooks are basically just wrappers for `auth.get`. Instead of a bunch of hooks, should we just have one (`useAuthValue`)?

Reply:
1. fixed! 
2. This is something that I've been thinking... I'm a bit worried that `useAuthValue` isn't as intuitive as `useAccessToken`, I understand the work for us might be a bit tedious but I can see that the hooks might be easier to understand by end users. What's your thought on this?

Reply:
well, i reverted the change to #1 @wjhsf as I noticed that without useLocalstorage, we need to make the auth data observables to trigger state update. Gonna do that in another PR.